data/015/1009/en_US.json

```diff
@@ -3,11 +3,11 @@
 		"1009": {
 			"name": "Unknown",
 			"message": "Miiverse functions are unavailable\non this system",
-			"short_description": "Unknown cause.",
-			"long_description": "Unknown cause.",
-			"short_solution": "Unknown solution.",
-			"long_solution": "Unknown solution.",
+			"short_description": "Your console has been permanently banned from Juxtaposition.",
+			"long_description": "Your console has been permanently banned from Juxtaposition.\n\nThis typically occurs because of a violation of the Code of Conduct, or other offenses on the Network or the Discord server.",
+			"short_solution": "Don't do whatever you did to get banned.",
```

Comment:
Same as the other ban message - this should reflect the long solution telling them to appeal